### PR TITLE
Added logic to show pIQ balance

### PR DESCRIPTION
**Pull request by Adebesin-Cell**
Added logic to show pIQ balance

fixes https://github.com/EveripediaNetwork/issues/issues/962

### CODE REVIEW

File: src/config/alchemy-sdk.ts [hunk 1]
```diff
@@ -11,5 +11,11 @@ const polygonNetworkSettings = {
...
 }
 
+const pTokenNetworkSettings = {
+  apikey: config.alchemyApiKey,
+  network: Network.ETH_MAINNET,
+}
+
 export const ethAlchemy = new Alchemy(ethNetworkSettings)
 export const polygonAlchemy = new Alchemy(polygonNetworkSettings)
+export const ptokenAlchemy = new Alchemy(pTokenNetworkSettings)
```
Comment (Emmanueldmlr) [inline src/config/alchemy-sdk.ts:21]: You dont need to create another instance for ptoken. eth instance works for it

File: src/pages/dashboard/bridge.tsx [hunk 2]
```diff
@@ -64,8 +64,11 @@ const Bridge: NextPage = () => {
     pIQBalance,
     bridgeFromEthToEos,
     bridgeFromPTokenToEth,
+    pIQTokenBalance,
   } = useBridge()
 
+  console.log(pIQTokenBalance)
```
Comment (Emmanueldmlr) [inline src/pages/dashboard/bridge.tsx:70]: remove console.log

File: src/hooks/useBridge.ts [hunk 3]
```diff
@@ -118,6 +120,8 @@ export const useBridge = () => {
   }
 
   return {
+    pIQTokenBalance:
+      parseInt(usePTokensBalance() as string, 10) / NORMALIZE_VALUE,
```
Comment (Emmanueldmlr) [inline src/hooks/useBridge.ts:124]: use formatContractResult function we have to handle this.

**Review by Emmanueldmlr — CHANGES_REQUESTED** [review]

File: src/config/alchemy-sdk.ts (hunk 1 above)
Comment (Adebesin-Cell) [reply src/config/alchemy-sdk.ts:21]: Ha! I'll fix it.

**Comment by Emmanueldmlr** [comment]
@Adebesin-Cell , I already fixed the error...Try to format the value and ensure you are displaying the right dollar value and not the token quantity.

**Comment by Adebesin-Cell** [comment]
> @Adebesin-Cell , I already fixed the error...Try to format the value and ensure you are displaying the right dollar value and not the token quantity.

Ha 🥲, thanks 🚀.

File: src/config/alchemy-sdk.ts (hunk 1 above)
Comment (Emmanueldmlr) [reply src/config/alchemy-sdk.ts:21]: this is still there @Adebesin-Cell ...remove the ptokenAlchemy

**Review by Emmanueldmlr — CHANGES_REQUESTED** [review]

**Comment by Adebesin-Cell** [comment]
@Emmanueldmlr Fixed.

**Review by Emmanueldmlr — APPROVED** [review]

**Comment by Adebesin-Cell** [comment]
@Emmanueldmlr I think we can merge now, right?

**Pull request merged**